index.bs

```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
+
+Design CSS properties or CSS layout systems (which are typically values of the `display` property), 
+to preserve the content as readable and usable by default.  
+
+<div class="example">
+For example, the default behavior of a layout system should not lead to content being clipped, 
```

Comment:
This doesn't seem like an example to me. Do we have specific examples of layout systems which demonstrate this principle?

Reply:
Well, we've kept this principle true in CSSWG over the years despite not having it written out explicitly. I'll change the wording to say that: "For example, the default behavior of all layout systems in CSS will not lead to content being clipped, content overlapping other content, or content being unreachable by scrolling. "